arch/arm/boot/dts/overlays/README

```diff
@@ -2858,6 +2858,8 @@ Params: poe_fan_temp0           Temperature (in millicelcius) at which the fan
                                 speeds up (default 55000)
         poe_fan_temp3_hyst      Temperature delta (in millicelcius) at which
                                 the fan slows down (default 5000)
+        i2c                     Control the fan via Linux I2C drivers instead of
+                                the firmware.
 
 
 Name:   rpi-poe-plus
```

Comment:
`rpi-poe-plus` inherits the `i2c` parameter, so also needs a README addition.

Reply:
I was looking at PoE+ with the power driver as well, but yes.